src/ffigen/ffigen.py

```diff
@@ -51,79 +51,190 @@ def read_xml_from_file(filename):
     "long unsigned int": "c_ulong",
     "long long int": "c_longlong",
     "long long unsigned int": "c_ulonglong",
+    "bool": "c_bool",
+    "float": "c_float",
+    "void": "void",
+    "uint64_t": "u64",
```

Comment:
Why isn't this "c_u64"?

Reply:
You're right, we mixed the enum name with the type name. Either way, this is no longer required with the new implementation, because `uint64_t` is just a typedef, and typedefs get resolved to the underlying type (`unsigned long long` in this case).